docs/components/modals/test.html

```diff
@@ -0,0 +1,431 @@
+          bears chocolate bar. Chocolate bear claw chocolate sweet roll marzipan.
+          Gummi bears cake macaroon muffin marshmallow. Candy sesame snaps donut
+          dragee danish tiramisu apple pie. Icing bear claw wafer chocolate cake
+          liquorice donut. Bonbon dragee bonbon candy canes bear claw jelly-o
+          jelly beans. Cotton candy wafer sugar plum caramels tootsie roll
+          chocolate bar macaroon cake. Gingerbread bear claw bonbon. Oat cake
+          jelly-o jelly-o danish candy jujubes. Sesame snaps chocolate brownie.
+          Pudding powder danish chocolate cake powder pudding sweet roll powder
+          cake. Bear claw pastry chocolate. Bear claw pastry chocolate. Bear.
+        </p>
+      </hx-div>
+      <footer>
+        <button class="hxBtn hxPrimary">Confirm</button>
+        <button class="hxBtn">Cancel</button>
+      </footer>
+    </hx-modal>
+  </section>
+
+  <section>
+    <h2>Modal with Positioned Elements</h2>
+    <hx-disclosure aria-controls="modal-with-positioned-elements" class="hxBtn">
+      Open Modal
+    </hx-disclosure>
+
+    <hx-modal id="modal-with-positioned-elements">
+      <header>
+        <h3>Modal Header</h3>
+      </header>
+      <hx-div>
+        <p>
+          <hx-disclosure class="hxBtn" aria-controls="modalMenuId">
+            <hx-icon type="cog"></hx-icon>
+            <span>Actions</span>
+            <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+          </hx-disclosure>
+          <br />
+          <hx-disclosure aria-controls="modalPopover" class="hxBtn">
+            Toggle Popover
+          </hx-disclosure>
+          <br />
+          <hx-icon data-tooltip="tooltip1" type="help-circle"></hx-icon>
+        </p>
+        <hx-menu id="modalMenuId">
+          <hx-menuitem>Action 1</hx-menuitem>
+          <hx-menuitem>Action 2</hx-menuitem>
+          <hx-menuitem>Action 3</hx-menuitem>
+        </hx-menu>
+
+        <hx-popover id="modalPopover" position="bottom-right">
+          <header>
+            Popover Header
+          </header>
+
+          <hx-div>
+            <p>
+              This is my popover body.
+            </p>
+          </hx-div>
+
+          <footer>
+            <button class="hxBtn hxPrimary">Submit</button>
+            <button class="hxBtn">Cancel</button>
+          </footer>
+        </hx-popover>
+
+        <hx-tooltip id="tooltip1" position="top">
+          Top
+        </hx-tooltip>
+      </hx-div>
+    </hx-modal>
+  </section>
+
+  <section>
+    <h2>Scrolling Modal with Positioned Elements</h2>
+    <hx-disclosure aria-controls="scrolling-modal-with-positioned-elements" class="hxBtn">
+      Open Modal
+    </hx-disclosure>
+
+    <hx-modal id="scrolling-modal-with-positioned-elements">
+      <header>
+        <h3>Modal Header</h3>
+      </header>
+      <hx-div scroll="vertical">
+        <p>
+          <hx-disclosure class="hxBtn" aria-controls="modalMenuId2">
```

Comment:
consider `modalMenuId2` -> `modal-menu-id2`

Reply:
@100stacks id like to keep consistent with the naming convention already in the code base :) maybe something we can refactor all-together in the future? any thoughts @CITguy ?

Reply:
I'll leave that decision to @cathysiller.

Reply:
for now ill keep it consistent with the demo id's and keep them as is -- NOTE: we should update and be consistent throughout the code base though, maybe something we can refactor on a free friday, we're doing different things in different files

Reply:
Ok no problem on my end @cathysiller @CITguy.  My suggestion stemmed from following the same naming convention you had for the other `ids` on this page.